use_case_examples/lora_finetuning/GPT2FineTuneHybrid.ipynb

```diff
@@ -846,7 +869,7 @@
     "tokenizer.parallelism = False\n",
     "\n",
     "# Train the model using FHE simulation\n",
-    "train_custom_model(hybrid_model, train_dataloader, training_args, tokenizer, fhe=\"simulate\")"
+    "train_custom_model(hybrid_model, train_dataloader, training_args, tokenizer, fhe=\"disable\")"
    ]
   },
```

Comment:
I don't understand why you are explicitly using the hybrid model, in: hybrid_model.model.inference_model

 you can use the CML lora API, no ?

Reply:
GPT2 is not using the LoraTrainer.